config/subsidy-application-management/forms/bicycle-infrastructure/request/versions/20210504170646/form.ttl

```diff
@@ -317,4 +317,4 @@ fieldGroups:1f3cf92c-aa5f-4708-85c6-77aabbc8989f form:hasField fields:94f3ab67-e
 fieldGroups:1f3cf92c-aa5f-4708-85c6-77aabbc8989f form:hasField fields:39b5f8a3-f163-4198-bf0d-82efeb2c0254 .
 fieldGroups:1f3cf92c-aa5f-4708-85c6-77aabbc8989f form:hasField fields:d91b1cc3-f2a1-4704-884c-e89f612cba33 .
 fieldGroups:1f3cf92c-aa5f-4708-85c6-77aabbc8989f form:hasField fields:1f17de22-e0c9-4166-ad32-a0d1dc11e375 .
-fieldGroups:1f3cf92c-aa5f-4708-85c6-77aabbc8989f form:hasField fields:2d154583-7e39-4bbf-9a53-ea7fd95e2c96 .
```

Comment:
This is an existing form, already submitted. I suppose we don't want to edit this one?

Reply:
Github marked it as 'changed' but there it a little red mark at the end saying: "No new line at end of file" when you hover over it so nothing really changed